src/auth/authenticator/tls/authenticator_test.go

```diff
@@ -57,7 +57,7 @@ func TestAuthenticatorSameCert(t *testing.T) {
 		assert.Equal(t, "eve", userInfo.Username)
 		assert.Equal(t, "auth0", userInfo.Tenant)
 		assert.Equal(t, []string{"signer"}, userInfo.Roles)
-		assert.Equal(t, types.ListWildcardPermission("*:eth1accounts"), userInfo.Permissions)
+		assert.Equal(t, types.ListWildcardPermission("*:ethereum"), userInfo.Permissions)
```

Comment:
Again, I am confused. Should it be `ethaccounts` ?

Reply:
why? `ethereum` is clearly a better name than `ethaccounts` since the domain is named `ethereum` so it correlates with keys and secrets